sinaloa/src/sinaloa_sgx.rs

```diff
@@ -33,14 +33,20 @@ pub mod sinaloa_sgx {
         sonora_sgx_ra_proc_msg2_trusted, sonora_start_local_attest_enc,
     };
     use std::{ffi::CStr, mem};
+    use std::io::Write;
+    use tempfile;
     use veracruz_utils;
 
     lazy_static! {
         static ref SONORA: std::sync::Mutex<Option<SgxEnclave>> = std::sync::Mutex::new(None);
     }
 
-    static MC_ENCLAVE_FILE: &'static str = "./target/debug/mexicocity.signed.so";
-    static SONORA_ENCLAVE_FILE: &'static str = "./target/debug/sonora.signed.so";
+    static MC_ENCLAVE_BINARY: &'static [u8] = include_bytes!(
+        "../../mexico-city-bind/target/debug/mexicocity.signed.so"
+    );
+    static SONORA_ENCLAVE_BINARY: &'static [u8] = include_bytes!(
```

Comment:
Is there already a PR to rename SONORA? Maybe match that name here.

Reply:
Can do, but I will wait for the Sonora PR to get merged first so there's less rebasing.